pydantic_ai_slim/pydantic_ai/profiles/anthropic.py

```
@@ -255,11 +306,12 @@ def anthropic_model_profile(model_name: str) -> ModelProfile | None:
         )
     )
     supported_native_tools = (
-        _ANTHROPIC_BASE_BUILTINS | {ToolSearchTool} if supports_tool_search else _ANTHROPIC_BASE_BUILTINS
+        _ANTHROPIC_BASE_BUILTINS | {ToolSearchTool} if supports_tool_search else _ANTHROPCI_BASE_BUILTINS
```

Ruff (0.15.21)

[error] Undefined name `_ANTHROPCI_BASE_BUILTINS`

(F821)